### ISSUE TITLE
react-native app with expo - unable to resolve es6.string.iterator

### ISSUE THREAD

**Issue by smankine** [jsonata-js/jsonata#255]
I'm trying to use jsonata with a react-native app but its failing when I'm running my app with

Unable to resolve "../../modules/es6.string.iterator" from "node_modules\jsonata\jsonata-es5.js"

Is it possible to use jsonata with react-native and particularly with expo?

**Comment by andrew-coleman** [comment]
I'm afraid I've no experience with react-native.  @asfordmatt, does anyone in your org use this with jsonata?

**Comment by andrew-coleman** [comment]
This should now be fixed in the latest release (v1.6.1).  Please could you re-test?
If you still see this problem, please re-open the issue and attach a code snippet that allows me to recreate.
Thanks

**Comment by Diwei-Chen** [comment]
Hi @andrew-coleman I just tested with v1.6.1 but it nows throws: 
```
bundling failed: Error: Unable to resolve module `./utils` from `.../node_modules/jsonata/jsonata.js`: The module `./utils` could not be found from `.../node_modules/jsonata/jsonata.js`. Indeed, none of these files exist:

  * `.../node_modules/jsonata/utils(.native||.ios.js|.native.js|.js|.ios.json|.native.json|.json)`
  * `.../node_modules/jsonata/utils/index(.native||.ios.js|.native.js|.js|.ios.json|.native.json|.json)`
```

FYI @adamgins

**Comment by andrew-coleman** [comment]
It's really hard to reverse engineer your environment from one error message.  Please could you post a link to some code with instructions that allow me to recreate this.  Otherwise I can't help.

If you are using the ES5 version of the library (jsonata-es5.js), then I have made fixes to this which have gone into v1.6.3.  Please try this first.
